src/loom-provider.ts

```diff
@@ -157,8 +158,8 @@ export class LoomProvider {
     )
 
     if (!this._setupMiddlewares) {
-      this._setupMiddlewares = (client: Client, privateKey: Uint8Array) => {
-        return createDefaultTxMiddleware(client, privateKey)
+      this._setupMiddlewares = (client: Client, privateKey: Uint8Array | null) => {
```

Comment:
Why is this `privateKey: Uint8Array | null` if you're assuming it's always `Uint8Array` on the line below?